src/options.ts

```diff
@@ -30,39 +30,41 @@ async function init() {
     await utils.wait(() => Config.config !== null);
 
     if (!showDonationLink()) {
-        document.getElementById("sbDonate").style.visibility = "hidden";
+        document.getElementById("sbDonate").style.display = "none";
```

Comment:
This was to prevent layout shifts

Reply:
I'm not sure what layout shifts you mean. That the "created by" text recenters itself? I thought that would be desirable. Especially because I moved things around a bit and it being off-center is more noticeable.

Reply:
The best solution I could come up with that wasn't off-center or pushing anything around was putting it into the corner:
![kép](https://user-images.githubusercontent.com/9072794/148690984-f04dbfc3-41db-4f9e-9719-f56c5bf8c523.png)